test/test_xcpp_kernel.py

```diff
@@ -131,6 +131,14 @@ class XCppTests(jupyter_kernel_test.KernelTests):
                 'mime': 'image/png'
             }
         ]
+        code_omp='#include <iostream>\n#include "omp.h"\n#include "clang/Interpreter/CppInterOp.h"\nCpp::LoadLibrary("libomp");omp_set_thread_num(4);#pragma omp parallel\n{\nif(omp_get_thread_num()==2) std::cout<<omp_get_thread_num()<<std::endl;}'
+        def test_xcpp_omp(self):
+            msg_id = self.kc.is_complete(code=self.code_omp)
+            reply = self.get_non_kernel_info_reply(timeout=1)
+            assert reply is not None
+            self.assertEqual(reply["msg_type"], "is_complete_reply")
+            self.assertEqual(reply["content"]["status"], "complete")
+            self.assertEqual(reply['content']['matches'], '2')  
```

Comment:
Is that enough to make sure that we spawn multiple threads?

Reply:
omp_set_thread_num(4) in the code will make sure we use use 4 threads for all parallel regions.

Reply:
I'm open to suggestions on what code you'd like me to try. At the moment I can't get the code I am trying to run to work, to even be able to test it. It just timeouts whenever I try and run my code, with no useful output as to what caused the timeout.

Reply:
@vgvassilev yes i'll look into an openmp test for CppInterOp